pleroma.py

```diff
@@ -75,7 +80,10 @@ def _unpack_id(obj):
 
 	async def status_context(self, id):
 		id = self._unpack_id(id)
-		return await self.request('GET', f'/api/v1/statuses/{id}/context')
+		try:
+			return await self.request('GET', f'/api/v1/statuses/{id}/context')
+		except BadResponse as exc:
+			raise exc
```

Comment:
revert this change. `except: raise` is identical to not having an exception block at all.